### DISCUSSION TITLE
Miro for students or teachers

### DISCUSSION THREAD

**Discussion by adrezpe** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#176] [Ideas]
Espero les pueda ser de ayuda.

Encontre que puedes crear una cuenta como estudiante en Miro lo cual te da muchas ventajas ya que se desbloquean algunas funciones interesantes y todas las plantillas que en la cuenta gratuita no puedes usar.

Les adjunto el enlace por si lo necesitan y les deseo mucho éxito explorers:

[Miro for students](https://miro.com/education-whiteboard/2/#:~:text=Miro%20provides%20a%20free%20education%20account%20to%20faculty,as%20long%20as%20you%20qualify%20for%20the%20license.)

**Comment by ArmandoRiveraT** [comment]
Muchas gracias por el aporte
Como dato, si le dan clic al link no funciona, tienen que copiarlo y pegarlo en una nueva ventana

**Reply by adrezpe** [reply]
Gracias, ya corregi el link para que se pueda abrir directamente ✌

**Comment by davidtoString** [comment]
Y se puede solicitar sin una cuenta de mail .edu?
